jobqueue/scheduler/scheduler_test.go

```diff
@@ -52,27 +52,28 @@ func init() {
 }
 
 func TestLocal(t *testing.T) {
+	ctx := context.Background()
```

Comment:
Somewhere there should be a test that our logs include the schedulertype context.